backend/models/Account.js

```diff
@@ -33,4 +33,4 @@ let Account = new Schema({
    }
 });
 
-export default mongoose.model('Account', Account);
\ No newline at end of file
+module.exports = mongoose.model('Account', Account);
```

Comment:
When refactoring the routes/controllers these imports had to be changed since the previous export way did not want to work